test/itkMontageTestHelper.hxx

```diff
@@ -151,12 +151,12 @@ int montageTest(const PositionTableType& stageCoords, const PositionTableType& a
         result = EXIT_FAILURE;
         }
       // write generated mosaic
-      ImageTypePointer image = montage->ResampleIntoSingleImage(false, 123.456);
```

Comment:
We should put the montaging into a Filter so that the filter can manage the output memory and so that we can stream the generation of large montages.

Reply:
This can be a next step.

Reply:
https://github.com/InsightSoftwareConsortium/ITKMontage/issues/46